documentation/src/docs/pravega-concepts.md

```diff
@@ -376,64 +271,56 @@ The following figure depicts the components deployed by Pravega:
+are usually multiple (recommended at least 3) Controller instances running in a running in a cluster for high availability.  
+
+The [Segment Store](Segment-store-service.md) implements the Pravega data plane.
+Pravega Servers provide the API to read and write data in Streams. Data storage is comprised of two tiers:
+- **Tier 1:** It provides short term, low-latency, data storage, guaranteeing the durability of data written to Streams. Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
+Tier 1 Storage. Tier 1 Storage typically runs _within_ the Pravega cluster. Tier 2 Storage is normally deployed _outside_ the Pravega cluster. Tiering storage is important to deliver the combination of fast access to Stream
+data and also allows large data storage for Streams. Tier 1 Storage
+persists on the most recently written Stream data. As data in Tier 1 Storage ages,
 it is moved into Tier 2 Storage.
 
+- **Tier 2:** It provides long term storage for Stream data. Pravega uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud Storage (ECS) to implement Tier 2 Storage.
+
+
 Pravega uses [Apache Zookeeper](https://zookeeper.apache.org/) as the
 coordination mechanism for the components in the Pravega cluster.  
 
-Pravega is built as a data storage primitive first and foremost.  Pravega is
-carefully designed to take advantage of software defined storage so that the
+Pravega is built as a data storage primitive first and foremost. Pravega is
+carefully designed to take advantage of software-defined storage, so that the
 amount of data stored in Pravega is limited only by the total storage capacity
-of your data center.  And like you would expect from a storage primitive, once
-data is written to Pravega it is durably stored.  Short of a disaster that
-permanently destroys a large portion of a data center, data stored in Pravega is never
-lost.
+of the data center. Once an Event is written to Pravega, it is durably stored and replicated so it can survive permanent crashes of datacenter nodes.
 
-Pravega provides a client library, written in Java, for building client-side
+Pravega provides a **Java Client Library**, for building client-side
 applications such as analytics applications using Flink. The Pravega Java Client
-Library manages the interaction between application code and Pravega via a
-custom TCP wire protocol.
+Library manages the interaction between the application code and Pravega via a
+custom TCP Wire Protocol.
+
+Storage tiering allows Pravega to achieve a sweet spot in the latency vs throughput trade-off. This makes Pravega an ideal storage substrate for serving data to both real-time and batch (analytics) applications. Moreover, as data in Tier 1 Storage ages, it is automatically moved into Tier 2 Storage. Thus, Pravega can store vasts amounts of Stream data and applications can read it at any time, while being oblivious to its actual location.
```

Comment:
This paragraph does not belong here, is part from a previous comment.

Reply:
Please check this